### ISSUE TITLE
[vue-docgen-api] Events defined in short `defineEmits` form are ignored

### ISSUE THREAD

**Issue by rijenkii** [vue-styleguidist/vue-styleguidist#1598]
**Current behavior**

When events are defined with [new short syntax](https://blog.vuejs.org/posts/vue-3-3#more-ergonomic-defineemits), they are not emitted by vue-docgen-api.

**To reproduce**

```vue
<script setup lang="ts">
const emit = defineEmits<{
  /** EventA docstring. */
  (evt: "eventA"): void;
  /** EventB docstring. */
  eventB: [];
}>();
</script>
```

```javascript
> const { parse } = await import("vue-docgen-api")
> const meta = await parse("./Comp.vue")
> console.log(meta)
{
  exportName: 'default',
  displayName: 'Comp',
  description: '',
  tags: {},
  expose: undefined,
  props: undefined,
  events: [
    {
      name: 'eventA',
      description: 'EventA docstring.',
      type: undefined
    }
  ],
  methods: undefined,
  slots: undefined,
  sourceFiles: [ './Comp.vue' ]
}
```

**Expected behavior**

Expected `meta.events` to contain description of `eventB`.

**Comment by crystalfp** [comment]
Other examples:
```
/** To communicate with the parent component */
const emit = defineEmits<{"close-panel": []}>();

const emit = defineEmits<{
    /** Return the edited values */
    changed: [value: Record<string, string | boolean | number>];

    /** Update the visibility of the dialogs */
    "update:modelValue": [visibility: boolean];
}>();
```
Thanks for looking!
mario

**Comment by rijenkii** [comment]
This was fixed by #1616.
